codebuild/scripts/deployment_variables.sh

```diff
@@ -0,0 +1,12 @@
+RELEASE_BUCKET_NAME_FMT="%s-%s"
```

Comment:
🎉🎉🎉 love this file thx.

nit: Consider renaming to `deployment_constants` or `static_deployment_variables`